website/versioned_docs/version-2.6/api-typedoc.json

```diff
@@ -1276,10 +1276,6 @@
 																"type": "literal",
```

Comment:
I believe these `versioned_docs` are autogenerated so no need to change it here. Also, you are changing the history (this is version 2.6. and the other file is 2.7 while the current version is 2.8) which is not correct.